modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -104,10 +106,11 @@ spec:
     app: echoserver
 ----
 <1> Defines the service resource.
-<2> Specifies the name of the service.
+<2> Specifies the service name.
 
-. Deploy an ALB-backed `Ingress` resource:
+. Create a YAML file that defines the `Ingress` resource:
 +
+.Example `ingress-albo.yaml` file
```

Comment:
```suggestion
.Example `ingress-albo.yaml` file:
```

Reply:
@opayne1 @xenolinux I don't think we need a colon here, right? This is a heading.

Reply:
@mburke5678 That's right. Removed the colon